aggregators/1.0/modules/ROOT/pages/aggregators-module-reference.adoc

```diff
@@ -182,185 +175,163 @@ image::time-based-aggregator.png[Time based Aggregator]
 
 ==== Parameters
 
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Aggregator Name | String | The name of the aggregator to listen to. Once that aggregator releases its elements, the listener will be executed. Each listener can only reference one aggregator, and each aggregator can only be referenced by at most one listener. | | x
+| Aggregator Name | String | The name of the aggregator to listen to. Once that aggregator releases its elements, the listener is executed. Each listener can only reference one aggregator, and each aggregator can only be referenced by at most one listener. | | x
 | Include Timed Out Groups | Boolean | Indicates whether the listener should be triggered when a group is released due to a timeout. | false |
 |===
 
 == Aggregation Attributes
 
-Each time a message goes through an aggregation, some attributes with information about the aggregation are added to it.
+Each time a message goes through an aggregation, some attributes with information about the aggregation are added to the message.
 
 [%header%autowidth.spread]
 |===
 | Name | Type | Description
-| Aggregation ID | String | The ID from the group where the element was aggregated. If the aggregation strategy does not aggregate by group, this field will be an autogenerated value that is kept until the aggregation is released (as with group-based and time-based aggregators).
-| First Item Arrival Time | Date | The time when the first value was aggregated.
-| Last Item Arrival Time | Date | The time when the last value was aggregated.
+| Aggregation ID | String | The ID from the group where the element is aggregated. If the aggregation strategy does not aggregate by group, this field will be an autogenerated value that is kept until the aggregation is released (as with group-based and time-based aggregators).
+| First Item Arrival Time | Date | The time when the first value is aggregated.
+| Last Item Arrival Time | Date | The time when the last value is aggregated.
 | Is Aggregation Complete | Boolean | True if the aggregation is complete, False otherwise.
 |===
 
-== Async Versus Sync Aggregations
+== Synchronous versus Asynchronous Aggregations
 
-There are two kinds of triggers for an aggregation completion: Synchronous and Asynchronous. +
-As seen in the configurations, an aggregation can be considered complete based on a new element being added to the list (as when a max size is specified) or because some timeout or time period was completed. _This is important because the type of aggregation determines which chain of components to execute with the list of elements._
+An aggregation is complete based on a new element added to the list, as when a Max size is specified, or because some timeout or time period is complete. This is important because the type of aggregation determines which chain of components to execute with the list of elements.
```

Comment:
Is "Max size" syntax? If so, format; if not, consider "maximum size":
```suggestion
An aggregation is considered complete either when a new element is added to the list (such as when a maximum size is specified) or when a specified timeout occurs or time period is complete. This is important because the type of aggregation determines which chain of components to execute with the list of elements.
```